pgfinder/validation.py

```diff
@@ -11,6 +11,7 @@
 LOGGER = logging.getLogger(LOGGER_NAME)
 
 
+# FIXME: This function doesn't even use its argument!
```

Comment:
Good spot. :+1:

Reply:
As with above comment should probably use the default configuration value rather than hard coding.

Reply:
I'll also try to clean this up when I am fixing up docstrings!